simpeg_drivers/constants.py

```diff
@@ -385,68 +279,199 @@
     "z_norm": {
         "association": "Cell",
         "dataType": "Float",
-        "group": "Regularization",
+        "group": "Sparse/blocky model",
         "label": "Z-smoothness norm",
         "isValue": True,
         "parent": "mesh",
-        "property": None,
+        "property": "",
         "value": 2.0,
         "min": 0.0,
         "max": 2.0,
         "precision": 2,
         "lineEdit": False,
         "enabled": True,
     },
-    "reference_model": {
-        "association": ["Cell", "Vertex"],
-        "main": True,
-        "dataType": "Float",
-        "group": "Mesh and models",
-        "isValue": True,
-        "optional": True,
-        "enabled": False,
-        "parent": "mesh",
-        "label": "Reference",
-        "property": None,
-        "value": 0.0,
-    },
     "gradient_type": {
         "choiceList": ["total", "components"],
-        "group": "Regularization",
+        "group": "Sparse/blocky model",
         "label": "Gradient type",
         "value": "total",
         "verbose": 3,
     },
-    "lower_bound": {
-        "association": ["Cell", "Vertex"],
-        "main": True,
-        "dataType": "Float",
-        "group": "Mesh and models",
-        "isValue": True,
-        "parent": "mesh",
-        "label": "Lower bound",
-        "property": None,
+    "max_irls_iterations": {
+        "min": 0,
+        "group": "Sparse/blocky model",
+        "label": "Maximum IRLS iterations",
+        "tooltip": "Incomplete Re-weighted Least Squares iterations for non-L2 problems",
+        "value": 25,
+        "enabled": True,
+        "verbose": 2,
+    },
+    "starting_chi_factor": {
+        "group": "Sparse/blocky model",
+        "label": "IRLS start chi factor",
+        "enabled": True,
+        "value": 2.0,
+        "tooltip": "This chi factor will be used to determine the misfit threshold after which IRLS iterations begin.",
+        "verbose": 3,
+    },
+    "chi_factor": {
```

Comment:
The cooling schedule and optimization parameters seem more fundamental than the weights and norms.  Consider moving to the top?
![image](https://github.com/user-attachments/assets/cc1800f8-18e2-46e2-a366-e98507a2b369)

Reply:
I don't think so. Most users can stick with defaults.